src/AuthService/Startup.cs

```diff
@@ -140,9 +142,19 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
                         .AllowAnyHeader()
                         .AllowAnyMethod());
 
+            var rabbitMqConfig = Configuration
+                .GetSection(BaseRabbitMqOptions.RabbitMqSectionName)
+                .Get<RabbitMqConfig>();
```

Comment:
move this variable in class fields and it initialize once

Reply:
Fixed